tests/integration/test_alias.py

```diff
@@ -0,0 +1,93 @@
+import csv
+import shutil
+from traceback import print_exception
+
+import pytest
+from click.testing import CliRunner
+
+from cromshell.__main__ import main_entry as cromshell
+
+
+def run_cromshell_alias(workflow_id: str, alias_name: str, exit_code: int):
+    """Run cromshell alias using CliRunner and assert job is successful"""
```

Comment:
This could be refactored, down the road, to be more general, so that other integration tests could be done similarly.

Reply:
function place in conftest.py

Reply:
It doesn't look like fixtures work in the same way normal functions do, I had some difficulty calling them and passing inputs to the fixture/function. So instead, this function was made generic and added to a new file called utility_test_functions.py under the integration folder